tests/kml_test.py

```diff
@@ -186,6 +186,61 @@ def test_kml_parse(self) -> None:
         assert doc.ns == "None"
 
 
+class TestWriteKML(StdLibrary):
+    def test_write_kml_file(self) -> None:
+
+        doc = kml.KML(
+              ns="{http://www.opengis.net/kml/2.2}",
+              name="Vestibulum eleifend lobortis lorem.",
+              features=[
+                  Document(
+                      ns="{http://www.opengis.net/kml/2.2}",
+                      id="doc-001",
+                      target_id="",
+                      name="Vestibulum eleifend lobortis lorem.",
+                      features=[
+                          Placemark(
+                              ns="{http://www.opengis.net/kml/2.2}",
+                          ),
+                      ],
+                      schemata=[],
+                  ),
+              ],
+          )
+
+        file_path = KMLFILEDIR / "output.kml"
+        doc.write(file_path=file_path, prettyprint=True, xml_declaration=True)
+
+        assert file_path.is_file(), "KML file was not created."
+
+    def test_write_kmz_file(self) -> None:
+
+        doc = kml.KML(
+              ns="{http://www.opengis.net/kml/2.2}",
+              name="Vestibulum eleifend lobortis lorem.",
+              features=[
+                  Document(
+                      ns="{http://www.opengis.net/kml/2.2}",
+                      id="doc-001",
+                      target_id="",
+                      name="Vestibulum eleifend lobortis lorem.",
+                      features=[
+                          Placemark(
+                              ns="{http://www.opengis.net/kml/2.2}",
+                          ),
+                      ],
+                      schemata=[],
+                  ),
+              ],
+          )
+
+        file_path = KMLFILEDIR / "output.kmz"
+
+        doc.write(file_path=file_path, prettyprint=True, xml_declaration=True)
+
+        assert file_path.is_file(), "KMZ file was not created."
```

Comment:
check that the `kml` is indeed inside the `kmz` file with name of `doc.kml` and the expected contents

Reply:
Added this check for both the test cases and deleted the output files